src/lib/isBefore.js

```diff
@@ -1,9 +1,11 @@
-import assertString from './util/assertString';
 import toDate from './toDate';
 
-export default function isBefore(str, date = String(new Date())) {
-  assertString(str);
-  const comparison = toDate(date);
-  const original = toDate(str);
-  return !!(original && comparison && original < comparison);
+export default function isBefore(date, options) {
```

Comment:
Yeah, you might be right. But I am of the school of clear/meaningful naming. However, I understand that this introduces inconsistencies. Not sure how much it would/could affect others, changing it here. Of course, this would just be a start to make wording/naming in the rest of the code base more consistent and precise.

When you read this function the mental parsing is just faster when it is immediately clear that it is a date (though it is not true, it is actually a date string) and the other prop is a date for comparison. TS would be so nice 😅. 

What we have now is a `str` and a `date`, not ideal. Just my opinion, though.

Reply:
Yeah, it doesn't really change anything so I would be fine with using `date` here. I have updated it for `isAfter`. I think we should keep `str` in the README and see later how we can improve that table.

And totally agree about TypeScript, but let's take this one step at a time.